frontend/src/pages/NewPipelineVersion.test.tsx

```diff
@@ -118,7 +118,7 @@ describe('NewPipelineVersion', () => {
     it('creates pipeline version is default when landing from pipeline details page', () => {
       tree = shallow(
         <TestNewPipelineVersion
-          {...generateProps(`?${QUERY_PARAMS.pipelineId}=${MOCK_PIPELINE.id}`)}
+          {...generateProps(`?${QUERY_PARAMS.pipelineId}=${MOCK_PIPELINE.pipeline_id}`)}
```

Comment:
IIRC, this change is unrelated to the change--which seems to be only removing a name check? 
So question was/is this test actually working?

Reply:
Great question! The short answer is: Yes, they are working. 

The `MOCK_PIPELINE.id` caused `pipelineId` to be `undefined`; however, those unit tests are validating the behavior which is independent to the `pipelineId`. Thus, the tests work even though we accidentally forget to fix it.